overrides/config/gregtech/worldgen/fluid/void/infinite_raw_oil_deposit.json

```diff
@@ -1,6 +1,6 @@
 {
   "weight": 100,
-  "name": "Void Infinite Raw Oil Deposit",
+  "name": "Void Infinite Oil Deposit",
   "yield": {
     "min": 80,
     "max": 80
```

Comment:
These should both be 40 to keep the overall oil product yield same as before. But it doesn't really matter